python/dask_cudf/dask_cudf/tests/test_accessor.py

```diff
@@ -24,12 +24,12 @@ def data_dt_2():
 
 
 @pytest.mark.parametrize("data", [data_dt_2()])
-@pytest.mark.xfail(raises=AttributeError)
 def test_datetime_accessor_initialization(data):
     pdsr = pd.Series(data.copy())
     sr = Series(pdsr)
     dsr = dgd.from_cudf(sr, npartitions=5)
-    dsr.dt
+    with pytest.raises(AttributeError):
+        dsr.dt
```

Comment:
Why do we raise an AttributeError here? Does the datetime accessor not work as intended?

Reply:
The data in `dsr` is of not Datetime type. It's a float Series.